package.json

```diff
@@ -39,28 +54,43 @@
     "release.docs": "sl-scripts release:docs",
     "release.dryRun": "sl-scripts release --dry-run --debug",
     "test": "jest",
-    "test.prod": "yarn lint && yarn test --coverage --maxWorkers=2",
+    "test.prod": "yarn build && yarn lint && yarn test --coverage --maxWorkers=2",
     "test.update": "yarn test --updateSnapshot",
     "test.watch": "yarn test --watch"
   },
   "dependencies": {
+    "@oclif/command": "^1",
+    "@oclif/plugin-help": "^2",
+    "@stoplight/json-ref-resolver": "^1.2",
```

Comment:
Suggest pegging stoplight deps to major/minor e.g. `1.2.x` (same for json and types).